test/manipulation.test.js

```diff
@@ -1675,7 +1675,7 @@ describe('manipulation', function() {
         db.automigrate('CustomModel5', done);
       });
 
-      it('should generate a new id when "defaultfn" is "uuid"', function(done) {
+      it('should generate a new id when "defaultFn" is "uuid"', function(done) {
```

Comment:
can you change the capital F back to lowercase f -- to remove this unrelated change.